exporter/opentelemetry-exporter-jaeger/src/opentelemetry/exporter/jaeger/__init__.py

```diff
@@ -140,13 +142,20 @@ def __init__(
             env_variable=Configuration().EXPORTER_JAEGER_PASSWORD,
             default=None,
         )
+        self.udp_split_oversized_batches = _parameter_setter(
+            param=udp_split_oversized_batches,
+            env_variable=Configuration().EXPORTER_JAEGER_AGENT_SPLIT_OVERSIZED_BATCHES,
```

Comment:
Is this env variable defined in the spec? If not it would be great to open a PR. I didn't see it looking here: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sdk-environment-variables.md#jaeger-exporter